app/models/turbo/streams/tag_builder.rb

```diff
@@ -121,7 +121,7 @@ def extract_target_name_from(target)
     def render_record(possible_record)
       if possible_record.respond_to?(:to_partial_path)
         record = possible_record
-        @view_context.render(partial: record.to_partial_path, locals: { record.model_name.singular.to_sym => record }, formats: [ :html ])
+        @view_context.render(partial: record.to_partial_path, locals: { record.model_name.param_key.to_sym => record }, formats: [ :html ])
```

Comment:
```suggestion
        @view_context.render(partial: record.to_partial_path, object: record, formats: :html)
```

Can we just pass `object` instead and let the partial name dictate the variable name? Also don't think we need the wrapping array around the format.

Reply:
💯, way better